tests/test_common.c

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file test_common.h
+ * @author Irfan
+ * @brief common header file for all tests to facilitate uniform logging format
+ *
+ * @copyright
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+#define _GNU_SOURCE
+
+#include <pthread.h>
+#include <assert.h>
+#include <stdio.h>
+#include <time.h>
+#include "sysrepo.h"
+#include "test_common.h"
+
+static void _test_log_msg(sr_log_level_t level, const char *message, const char* prefix)
```

Comment:
Coding style, in definitions the return value is always on a separate line. For declarations, everything is on a single line (unless wrapped, of course).